reference/create-index-using-pgroonga.md

```diff
@@ -96,6 +96,30 @@ See [How to customize token filters](#custom-token-filters) for token filters de
 
 Specify `tokenizer='${TOKENIZER_NAME}'` for customizing tokenizer. Normally, you don't need to customize tokenizer.
 
+**NOTE: Partial match in alphabetic languages**
```

Comment:
How about creating a new `#### ` level section after `#### How to customize tokenizer` instead of adding a note in `#### How to customize tokenizer`?

Reply:
OK!

I move that note part to new `#### ` level section!